src/Symfony/Bundle/SecurityBundle/DependencyInjection/Compiler/CustomAccessDecisionManagerPass.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Symfony\Bundle\SecurityBundle\DependencyInjection\Compiler;
+
+
```

Comment:
These 2 double empty lines should become just one empty line. You should also include the license header between line 1 and 3 as you can see in other files in the code base.